assemblies/rosa-install-logging.adoc

```diff
@@ -0,0 +1,29 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-install-logging"]
+:context: rosa-install-logging
+toc::[]
+
+= Installing logging add-on services {product-title}
+
+[role="_abstract"]
+Installing the logging add-on and Amazon Web Services (AWS) CloudWatch log forwarding add-on services on {product-title} (ROSA).
+
+The AWS CloudWatch log forwarding service on ROSA has the following approximate log throughput rates (per worker node). These rates can result in dropped log messages.
```

Comment:
Instead of "These rates can result in dropped log messages"

"Message rates greater than this may result in dropped log messages"

Reply:
Done.